packages/postgres-database/tests/conftest.py

```diff
@@ -283,6 +291,35 @@ async def _creator(conn, user: RowProxy, **overrides) -> RowProxy:
         )
 
 
+@pytest.fixture
+def create_fake_uuid(faker: Faker) -> Callable[[], uuid.UUID]:
+    def _creator() -> uuid.UUID:
+        random_uuid = faker.uuid4(cast_to=None)
```

Comment:
why not just `faker.uuid4(cast_to=None)`?  In any case, if you prefer this helper I recommend to move it to some type of plugin in `pytest_simcore` that extends `fake` e.g. `pytest_simcore.faker_extension` ?

Reply:
I find cumbersome I have to put an assert to check the type everytime I use it. otherwise I could just not use the faker.
but I agree to move it elsewhere

Reply:
ok I removed it and I use directly ```uuid.uuid4()``` instead